index.html

```diff
@@ -650,18 +650,18 @@ <h2 id="with-sd-jwt">With SD-JWT</h2>
         <section>
             <h2 id="securing-with-sd-jwt">Securing JSON-LD Verifiable Credentials with SD-JWT</h2>
             <p>
-                This section details how to use JOSE to secure verifiable credentials conforming
+                This section details how to use JOSE to secure [=verifiable credentials=] conforming
                 to [[VC-DATA-MODEL-2.0]].
             </p>
             <p>
                 A [=conforming SD-JWT issuer implementation=] MUST use [[[SD-JWT]]] [[SD-JWT]] to secure
                 this media type. The unsecured [=verifiable credential=] is the input JSON
                 claim set. The Issuer then converts the input JSON claim set (i.e., the
-                unsecured [=verifiable credential=]) into an SD-JWT payload according to
+                unsecured [=verifiable credential=]) into an [[SD-JWT]] payload according to
                 <a data-cite="SD-JWT#section-6.1">SD-JWT issuance instructions</a>.
             </p>
             <p>
-                The <code>typ</code> header parameter SHOULD be <code>vc-ld+sd-jwt</code>.
+                The <code>typ</code> header parameter SHOULD be <code>vc+sd-jwt</code>.
                 When present, the <code>cty</code> header parameter SHOULD be <code>vc</code>.
```

Comment:
```suggestion
                The <code>cty</code> header parameter MUST be <code>vc</code>.
```

Reply:
David, since this suggestion touches on language not proposed in this PR, please raise a separate issue to discuss your desire to strengthen the normative language here. The group agreed to use the 'cty' property, it was not decided how strongly to require it.

Reply:
"agreed to use it" to differentiate between our use of `typ` with the IETF's use, implies to me that`cty` was to be mandated. Otherwise, the current text implies it is both optional (i.e. When present) and even then only SHOULD be used. 

So I maintain that my proposed rewording is in the spirit of the WG resolution, and that the current text is definitely not in the spirit of the resolution. So I do not think it is appropriate to raise another issue, since the current PR is meant to be implementing the WG resolution.